repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java

```diff
@@ -211,12 +210,19 @@ private void checkToolsLoaded() {
     }
 
     private void checkNopolSolverPath() {
-        File file = new File(this.arguments.getString("z3Path"));
-
-        if (!file.exists()) {
-            System.err.println("The Nopol solver path should be an existing file: "+file.getPath()+" does not exist.");
+    	solverPath = arguments.getString("z3Path");
```

Comment:
I think that it happened because I created a new branch to work with before you merged.